README.md

```diff
@@ -56,9 +56,23 @@ $ make run-console-local
 
 Then visit `http://localhost:9000` to view the console.
 
+## Subscribe to a Package and Channel
```

Comment:
I feel the initial example should be in terms of `kubectl` and inline yaml instead of a UI.

The existing example was geared more towards showing how to spin up an OLM aware UI locally, rather than how to use OLM through it.

Reply:
It is something that already existed. Just organized in a subsection.